css/amp.css

```diff
@@ -126,6 +126,33 @@ i-amp-sizer {
   /* TODO(dvoytenko): explore adding here object-fit. */
 }
 
+/**
+ * Makes nodes visually invisible but still accessible to screen-readers.
+ *
+ * This is a public class so page authors can use it for similiar a11y purposes
+ * within their AMP documents on any element.
+ *
+ * This css has been carefully tested to ensure screen-readers can read and
+ * activate (in case of links and buttons) the elements with this class. Please
+ * use caution when changing anything, even seemingly safe ones. For example
+ * changing width from 1 to 0 would prevent TalkBack from activating (clicking)
+ * buttons despite TalkBack reading them just fine.
+ */
+.amp-screen-reader {
```

Comment:
this is not always a `div` ( actually most cases it would be a `button` or a `p`) and it is not always a new element that is created in a component, sometimes we want to toggle this class on an existing node (for example to fix https://github.com/ampproject/amphtml/issues/5487 we could just add this to the next/previous buttons when hiding them).

We can add `dom.js` utilities but we should keep this in Css (another reason to keep it in Css is that we want to expose this to page authors as part of the kick-start core Css)